src/postgres/commands/PostgresAccountWizard/PostgresServerCredPWStep.ts

```diff
@@ -15,7 +15,10 @@ export class PostgresServerCredPWStep extends AzureWizardPromptStep<IPostgresWiz
         const user = nonNullProp(wizardContext, 'adminUser');
         wizardContext.adminPassword = (await ext.ui.showInputBox({
             placeHolder: localize('pwPlaceholder', 'Password'),
-            prompt: localize('enterPWPrompt', 'Enter administrator password for the server.'),
+            prompt: localize('enterPWPrompt', 'Enter administrator password for the server. ' +
+                '**Note** Password must contain characters from three of the following categories: ' +
+                '"uppercase letters", "lowercase letters", "numbers (0-9)", and "non-alphanumeric characteries (!, $, etc.)"'
```

Comment:
nit: I know "characteries" is technically correct here (I didn't know it was a word until today lol) but "characters" makes more sense at least to me
```suggestion
                '"uppercase letters", "lowercase letters", "numbers (0-9)", and "non-alphanumeric characters (!, $, etc.)"'
```

Reply:
Lol its a typo 😅 I didn't know characteries was word, thanks for catching it